framework/logging.cpp

```diff
@@ -1598,12 +1693,23 @@ static int print_one_thread_messages(int fd, struct per_thread_data *data, struc
         std::string_view message(ptr, delim - ptr);
         switch (log_type_from_code(code)) {
         case UserMessages:
-            format_and_print_message(fd, -1, message);
+            format_and_print_message(fd, -1, message, true);
```

Comment:
Boolean trap... sorry, this was my idea to use the same function, but it didn't end up looking good. I'll figure out a way to fix this.